Content.Client/Inventory/StrippableBoundUserInterface.cs

```diff
@@ -29,74 +50,186 @@ protected override void Open()
         {
             base.Open();
 
-            var entMan = IoCManager.Resolve<IEntityManager>();
-            _strippingMenu = new StrippingMenu($"{Loc.GetString("strippable-bound-user-interface-stripping-menu-title", ("ownerName", Identity.Name(Owner.Owner, entMan)))}");
+            _entMan = IoCManager.Resolve<IEntityManager>();
+            _hud = IoCManager.Resolve<IGameHud>();
+            _protoMan = IoCManager.Resolve<IPrototypeManager>();
+            _resourceCache = IoCManager.Resolve<IResourceCache>();
+            _examine = _entMan.EntitySysManager.GetEntitySystem<ExamineSystem>();
+            _inv = _entMan.EntitySysManager.GetEntitySystem<InventorySystem>();
```

Comment:
With these long lines of resolves I realise it might be better to just get the dependency collection once and resolve from there.